VirtoCommerce.Platform.Web/Controllers/HomeController.cs

```diff
@@ -41,6 +50,8 @@ public ActionResult Index()
                 }
             }
 
+            ViewBag.FavIcon = GetFavIcon();
```

Comment:
maybe will be better extend `PlatformSetting` with new property `FavIcon` instead of define the new property in ViewBag? It will be more `by current design`, thoughts?

Reply:
Is seems strange to me to use Model in layout (as it could be used in many pages with different models - and we expect all of them to have some property).
In our case - layout is used in one page, so we know what model we could use.
In general i think ViewBag is more appropriate for this as it is dynamic collection initially, and model is strongly typed.
In case we are not going to create additional views with this layout - we could extend PlatformSetting.
What do you think?

Reply:
Or maybe we could use this approach:
https://stackoverflow.com/a/14882183
We will have strongly typed object then.

Reply:
You right, it is the absolutely not `by right way` solution  to access in Layout any of concrete view models, but in your case we have a SPA application that has  only one view and this works. Regarding of   question of using ViewBug or strongly typed ViewModel in layout the both cases are incorrect due that fact described above. Why I'm leaning towards the extending `PlatformSetting` type with new property? Because  this approach already exists and works, and these changes will be consistent with the current design.

Reply:
Ok, agreed, will extend PlatformSetting to be consistent with current approach.

Reply:
Done